receiver/apachesparkreceiver/metadata.yaml

```diff
@@ -4,4 +4,783 @@ status:
   class: receiver
   stability: 
     development: [metrics]
-  distributions: [contrib]
\ No newline at end of file
+  distributions: [contrib]
+
+attributes:
+  application_id:
+    description: The ID of the application for which the metric was recorded.
+    type: string
+  application_name:
+    description: The name of the application for which the metric was recorded.
+    type: string
+  stage_id:
+    description: The ID of the application stage for which the metric was recorded.
+    type: int
+  attempt_id:
+    description: The ID of the stage attempt for which the metric was recorded.
+    type: int
+  stage_status:
+    description: The status of the stage for which the metric was recorded.
+    type: string
+    enum:
+      - ACTIVE
+      - COMPLETE
+      - PENDING
+      - FAILED
+  source:
+    description: The source from which data was fetched for the metric.
+    type: string
+    enum:
+      - local
+      - remote
+  location:
+    description: The location of the memory which is quantified in the metric.
+    type: string
+    enum:
+      - on_heap
+      - off_heap
+  executor_id:
+    description: The ID of the executor for which the metric was recorded.
+    type: string
+  job_id:
+    description: The ID of the job for which the metric was recorded.
+    type: int
+  pool_memory_type:
+    description: The type of pool memory for which the metric was recorded.
+    type: string
+    enum:
+      - direct
+      - mapped
+  gc_type:
+    description: The type of the garbage collection performed for the metric.
+    type: string
+    enum:
+      - major
+      - minor
+
+metrics:
+  #stage
+  spark.stage.active_tasks:
```

Comment:
I think this should share a namespace with the other task related metric. e.g. `spark.stage.task.active`